otx/algorithms/action/configs/classification/movinet/template.yaml

```diff
@@ -0,0 +1,63 @@
+# Description.
+model_template_id: Custom_Action_Classificaiton_MoViNet
+name: MoViNet
+task_type: ACTION_CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Basic transfer learning template for MoViNet
+application: ~
+
+# Algo backend.
+framework: OTXAction v2.9.1
```

Comment:
it uses mmaction2 v0.24.1

Reply:
@jaegukhyun Do you have any concern about side effect by updating to `mmaction2 v0.24.1`? If not, I'll also update X3D template :)

Reply:
Yes, this parameter will be used in Geti

Reply:
This is the version for `OTXAction`, not `mmaction`. `2.9.1` doesn't seem appropriate.